src/components/media-object/README.md

```diff
@@ -1 +1,137 @@
+          </g>
+        </svg>
+      </button>
+      <div class="rvt-dropdown__menu rvt-dropdown__menu--right" id="dropdown-1" role="menu" aria-hidden="true">
+        <button type="button" role="menuitemradio">Notify all</button>
+        <button type="button" role="menuitemradio" aria-checked="true">Notify admins</button>
+        <button type="button" role="menuitemradio">Notify contributors</button>
+      </div>
+    </div>
+  </div>
+</div>
+```
+
+#### Combining media object and box components
+
+You can combine the media object component with the [box component](#) to create list views for complex content. This combination can be used in place of tables when displaying lists of records that are not tabular data.
+
+```
+<div class="rvt-box">
+  <div class="rvt-box__header">
+    Messages
+  </div>
+  <div class="rvt-box__row">
+    <div class="rvt-mo">
+      <div class="rvt-mo__prepend">
+        <div style="width: 2rem; height: 2rem; border-radius: 999rem; overflow: hidden;" class="rvt-m-right-sm">
+          <img src="https://www.fillmurray.com/g/200/200" alt="">
+        </div>
+      </div>
+      <div class="rvt-mo__main">
+        <a href="#" class="rvt-link-bold rvt-ts-18">Rivet 1.2.0 is now available!</a>
+        <div class="rvt-ts-14">Updated Sept. 24 by <strong>bmacklin</strong></div>
+      </div>
+    </div>
+  </div>
+  <div class="rvt-box__row">
+    <div class="rvt-mo">
+      <div class="rvt-mo__prepend">
+        <div style="width: 2rem; height: 2rem; border-radius: 999rem; overflow: hidden;" class="rvt-m-right-sm">
+          <img src="https://www.fillmurray.com/g/200/200" alt="">
+        </div>
+      </div>
+      <div class="rvt-mo__main">
+        <a href="#" class="rvt-link-bold rvt-ts-18">Rivet 1.2.0 is now available!</a>
+        <div class="rvt-ts-14">Updated Sept. 24 by <strong>bmacklin</strong></div>
+      </div>
+    </div>
+  </div>
+  <div class="rvt-box__row">
+    <div class="rvt-mo">
+      <div class="rvt-mo__prepend">
+        <div style="width: 2rem; height: 2rem; border-radius: 999rem; overflow: hidden;" class="rvt-m-right-sm">
+          <img src="https://www.fillmurray.com/g/200/200" alt="">
+        </div>
+      </div>
+      <div class="rvt-mo__main">
+        <a href="#" class="rvt-link-bold rvt-ts-18">Rivet 1.2.0 is now available!</a>
+        <div class="rvt-ts-14">Updated Sept. 24 by <strong>bmacklin</strong></div>
+      </div>
+    </div>
+  </div>
+</div>
+```
+
+## Implementation notes
+
+### Media object content areas
+
+Each of the three media object areas (`.rvt-mo__prepend`, `.rvt-mo__main`, `.rvt-mo__append`) will expand to fit the content you put inside of them, but you should try to maintain an approximate ratio where the the main content area (`.rvt-mo__main`) takes up the majority of the width of the row, and the two supplemental areas (`.rvt-mo__prepend` and `.rvt-mo__append`) fill the remaining space.
+
+#### Media objects and the grid
```

Comment:
```suggestion
### Media objects and the grid
```

This seems like more of a sub item of `Implementation notes` than `Media object content areas`.